sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosRowConverter.scala

```diff
@@ -19,8 +16,9 @@ import org.apache.spark.sql.catalyst.expressions.{GenericRowWithSchema, UnsafeMa
 import org.apache.spark.sql.catalyst.util.ArrayData
 
 import java.io.IOException
-import java.time.{Instant, LocalDate, OffsetDateTime, ZoneOffset}
+import java.sql.{Date, Timestamp}
```

Comment:
Please add corresponding Test coverage in CosmosRowConverterITest and CosmosRowConverterSpec - these were created by Matias - and are one of the best set of tests we have in all of the spark connector. Having extensive coverage of teh RowConverter functionality there has proven very useful

Reply:
ack, will do.